static/app/views/replays/utils/useReplayEvent.tsx

```diff
@@ -188,7 +188,7 @@ function useReplayEvent({eventSlug, location, orgId}: Options): Result {
     }
   }
 
-  useEffect(() => void loadEvents(), [orgId, eventSlug, retry]);
+  useEffect(() => void loadEvents(), [orgId, eventSlug, retry]); // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
It asked for `loadEvents` to be in the dependency list. That in turn requires all the earlier closures to use `useCallback()` which isn't the end of the world. The problem is some value isn't coming from a ref, so it would cause infinite re-renders.

Decided to skip the check here instead.